test/test_torch.py

```diff
@@ -197,6 +197,48 @@ def test_sinh(self):
     def test_lgamma(self):
         self._testMathByName('lgamma')
 
+    def test_digamma(self):
+        makeshift = True  # TODO Replace makeshift digamma() with higher precision version.
+        x = torch.Tensor([-0.1, 3, 999])
+        expected = torch.Tensor([
+            9.24507305005294,
+            0.92278433509846,
+            6.90625419464,
+        ])
+
+        out = torch.digamma(x)
+        self.assertEqual(out, expected, 1e-5)
+
+        out = x.digamma()
+        self.assertEqual(out, expected, 1e-5)
```

Comment:
Why not include those three values in the test that compares to scipy below?

Reply:
Leaving these here since I've compressed the values below into a `torch.logspace()`. Also the values below span a wider range and require a lower precision (since `TestCase.assertEqual()` does not deal well with relative error).